velox/exec/tests/PlanBuilderTest.cpp

```diff
@@ -97,23 +97,35 @@ TEST_F(PlanBuilderTest, windowFunctionCall) {
 
   registerWindowFunction();
 
-  VELOX_CHECK_NOT_NULL(
+  VELOX_CHECK_EQ(
       PlanBuilder()
           .tableScan(ROW({"a", "b", "c"}, {VARCHAR(), BIGINT(), BIGINT()}))
           .window({"window1(c) over (partition by a order by b) as d"})
-          .planNode());
+          .planNode()
+          ->toString(true, false),
+      "-- Window[partition by: [a] order by: [b ASC NULLS LAST] "
```

Comment:
it is a bit odd that 'partition by' are lower case while 'RANGE ...' are all caps

Reply:
It seemed like anything from an enum was uppercase, while the PlanNode field names are in lower case. I've now only left enums as upper case.